api/public_api_v1.yaml

```diff
@@ -237,33 +246,45 @@ definitions:
         description: Number of items returned for each page.
       next:
         type: string
-        description: 'Contains an URL to GET the next #<page_size> results in the retrieved collection of items.'
+        description: 'Contains an URL to GET the next #<page_size> results in the
```

Comment:
is the quote required? I would remove it for consistency with the rest

Reply:
it's required as the character "#" starts a yaml comment